modules/web/gen-default-nodes/index.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
We don't use the index.js in other places so shall we stick with a proper file name